biome.jsonc

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "https://biomejs.dev/schemas/2.1.4/schema.json",
+  "vcs": {
+    "enabled": false,
+    "clientKind": "git",
+    "useIgnoreFile": false
+  },
+  "files": {
+    "ignoreUnknown": false,
+    "includes": [
+      "**",
+      "!**/node_modules",
+      "!**/dist",
+      "!**/build",
+      "!**/coverage",
+      "!**/.git",
+      "!**/.idea",
+      "!**/.vscode",
+      "!**/.github",
+      "!**/public/notyf.min*",
+      "!**/schemas",
+      "!**/package.json",
+      "!**/package-lock.json"
+    ]
+  },
+  "formatter": {
+    "indentStyle": "space",
+    "indentWidth": 2,
```

Comment:
I suggest changing the tab width back to 4 to avoid touching everything needlessly. @veggiesaurus also suggested matching the existing prettierrc. Apparently there's an automated process for this: https://biomejs.dev/guides/migrate-eslint-prettier/

Reply:
Have migrated over the prettier config and applied the results.  Happy now?